stream_alert/sink.py

```diff
@@ -81,6 +84,66 @@ def _get_sns_topic_arn(self):
             topic=topic
         )
 
+    def _chunk_sns_messages(self):
+        """Return a list of base64 encoded SNS messages.
+
+        These are split to ensure each is less than 256kb in size.
+
+        Returns:
+            List of encoded messages that are sent to be published
+            one at a time.
+        """
+
+        chunks = []
+        sns_dict = {'default': []}
+
+        for i in range(len(self.alerts)):
+            added = False
+            sns_json_message = self._get_json_from_dict(sns_dict)
+
+            b64_message_size = self._calculate_b64_size(sns_json_message)
+
+            new_json_message =  ""
+
+            if b64_message_size < _SNS_MAX_SIZE:
```

Comment:
not <= because it wouldn't be able to get added to the message anyway and would fall through to the below statement (the smallest an alert can be is always > 1). will add comments